rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -265,20 +265,33 @@ describe("ActivityList", () => {
         return Promise.resolve({ data: {} });
       });
       const wrapper = await shallowMountActivityList();
+      await wrapper.vm.$nextTick();
       // Simulate clicking the filter button to show filters
       const filterButton = wrapper.find(
         '[data-testid="activity-list-filter-button"]',
       );
+      console.log("Filter button found:", filterButton.exists());
       await filterButton.trigger("click");
+      await wrapper.vm.$nextTick();
       expect(wrapper.vm.showFilters).toBe(true);
       // Simulate entering a search term
       const activityFilter = wrapper.findComponent(ActivityFilter);
-      activityFilter.vm.$emit("input", {
-        ...wrapper.vm.query,
-        jobIdFilter: "testJob",
-      });
```

Comment:
@smartinellibenedetti Can you please look at this please? By doing this test is passing. But when I started reviewing my code then I realized if I can call the ActivityFilter directly and look for the input. and set the input value and search it. While by doing this test is failing. Is this a better approach or just follow this steps ? Please advise if I am missing something. Thanks 
```
activityFilter.vm.$emit("input", {
        ...wrapper.vm.query,
        jobIdFilter: "testJob",
      });
```